packages/donation-form/src/donation-form.ts

```diff
@@ -228,31 +225,44 @@ export class DonationForm extends LitElement {
       return;
     }
 
+    const contactInfo = this.contactForm.donorContactInfo;
+
+    switch (this.selectedPaymentProvider) {
+      case PaymentProvider.CreditCard:
+        this.handleCreditCardDonationFlow(contactInfo);
+        break;
+      case PaymentProvider.Venmo:
+        this.handleVenmoDonationFlow(contactInfo);
+        break;
+    }
```

Comment:
what is the default? (should credit card be default?)

Reply:
I don't think we want a default for this since it kicks off a payment flow. This makes sure the user has chosen a payment option before we do anything. The user has to first choose a payment option before they can click on the Donate button so we'll always have an option chosen when it gets here.